sdcm/cluster.py

```diff
@@ -2047,6 +2047,8 @@ def offline_install_scylla(self, unified_package, nonroot):
             self.remoter.run('sudo update-java-alternatives --jre-headless -s java-1.8.0-openjdk-amd64')
 
         if nonroot:
+            self.remoter.run('env', verbose=True, change_context=True)
+            assert 'XDG_RUNTIME_DIR' in self.remoter.run('env', verbose=True).stdout
```

Comment:
@amoskong may it make sense to check first is there such env variable and if missing then reconnect?

Reply:
> @amoskong may it make sense to check first is there such env variable and if missing then reconnect?

Good idea, fixed.